contracts/contracts/ccip/onramp/messages.tolk

```diff
@@ -91,6 +91,10 @@ struct (0xac1dd12e) OnRamp_MessageValidationFailed<T> {
     context: T
 }
 
+struct (0x7052dc75) OnRamp_WithdrawFeeTokens {
+    feeTokens: SnakedCell<address>;
```

Comment:
We would also need amounts.

As this is an async env best option is the sender checks balances off-chain, sends amounts and we try to transfer to fee aggregator.